superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -182,7 +183,7 @@ export default function DataSourcePanel({
 
   const mainBody = (
     <>
-      <div className="field-selections" style={{overflowY: 'scroll', height: '88%'}}>
```

Comment:
Does this mean that a scrollbar no longer appears?

Reply:
No, There are already tags for overflow in the instantiation of the at Line 61, and adding Style attribute to tags is not recommended, so I went ahead and put my CSS changes where the rest of the CSS for class name "field-selections" are.